assets/css/style.css

```diff
@@ -0,0 +1,244 @@
+@import url("https://fonts.googleapis.com/css?family=Arvo:400,700,400italic");
```

Comment:
I think this file is automatically downloaded by Jekyll and should be included in the repo.  It's not in our internal docs repo.

Reply:
Yup. I have Removed this file now.